packages/preferences/src/browser/views/preference-searchbar-widget.tsx

```diff
@@ -53,19 +56,49 @@ export class PreferencesSearchbarWidget extends ReactWidget {
         }
     }
 
+    protected clearSearchResults = (e: React.MouseEvent): void => {
+        const search = document.getElementById(PreferencesSearchbarWidget.SEARCHBAR_ID) as HTMLInputElement;
+        if (search) {
+            search.value = '';
+            this.search(search.value);
+            this.update();
+        }
+    };
+
+    protected renderOptionContainer(): React.ReactNode {
+        const clearAllOption = this.renderOptionElement();
+        const resultsCount = this.renderResultsCount();
+        return <div className="option-buttons"> {this.searchbarRef.current?.value !== '' ? resultsCount : ''} {clearAllOption} </div>;
+    }
+
+    protected renderResultsCount(): React.ReactNode {
+        return <span
+            className="results-found"
+            title={`${this.searchCount} Settings Found`}> {this.searchCount == 0 ? 'No' : this.searchCount} Settings Found </span>;
```

Comment:
The following line currently fails the build (eslint rule), we should use the triple equals.

```suggestion
            title={`${this.searchCount} Settings Found`}> {this.searchCount === 0 ? 'No' : this.searchCount} Settings Found </span>;
```

For information, the triple equals checks for type equality along with value equality.